File: src/led/Area.java
package led;

public class Area {
	int mStartIndex;
	int mEndIndex;
	
	public Area(int _start, int _end) {
		mStartIndex = _start;
		mEndIndex = _end;
	}
	
	public Area(Area _area) {
		mStartIndex = _area.mStartIndex;
		mEndIndex = _area.mEndIndex;
	}
	
	public void redefineArea(int _start, int _end) {
		mStartIndex = _start;
		mEndIndex = _end;
	}
	
	public int getStartIndex() { return mStartIndex; }
	public int getEndIndex() { return mEndIndex; }
}

File: src/main/java/led/Area.kt
package led

class Area {
    var startIndex: Int
    var endIndex: Int

    constructor(_start: Int, _end: Int) {
        startIndex = _start
        endIndex = _end
    }

    constructor(_area: Area) {
        startIndex = _area.startIndex
        endIndex = _area.endIndex
    }

    fun redefineArea(_start: Int, _end: Int) {
        startIndex = _start
        endIndex = _end
    }
}